discord/ext/menus/__init__.py

```diff
@@ -0,0 +1,10 @@
+"""
+discord.ext.menus
+~~~~~~~~~~~~~~~~~~~~~
+An extension module to provide useful menu options.
+
+:copyright: 2021 Pycord-Development
```

Comment:
i forgot to add present lmao
```suggestion
:copyright: 2021-present Pycord-Development
```